src/components/VmsList/style.css

```diff
@@ -61,10 +61,6 @@
     z-index: 20;
 }
 
-.full-window {
```

Comment:
Did this change impact the fix or is it just cleanup?

Is the class used in Vms.js?  Does it need to be anymore?  Perhaps remove it from there as well.

Reply:
It is just cleanup.

Reply:
our github standards are complete yet, but single-purpose commits are definitely in there :) so next time let's cut cleanups into separate commits and PRs